src/main/java/com/itschool/eventmanagment/services/EventServiceImp.java

```diff
@@ -1,10 +1,21 @@
 package com.itschool.eventmanagment.services;
 
+import com.itschool.eventmanagment.models.dtos.DetailsParticipantDTO;
 import com.itschool.eventmanagment.models.dtos.EventDTO;
+import com.itschool.eventmanagment.models.dtos.ParticipantDTO;
+import com.itschool.eventmanagment.models.entities.Event;
+import com.itschool.eventmanagment.models.entities.Participant;
+import com.itschool.eventmanagment.repositories.EventRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
 
-import java.util.List;
+import java.util.*;
 
+@Service
 public class EventServiceImp implements EventService {
+    @Autowired
+    private EventRepository eventRepository;
```

Comment:
In the future, we could have other dependencies like `ObjectMapper` or other services. Maybe we can start to inject the dependencies into the constructor?
There is nothing wrong with dealing with Dependency Injection with the help of `@Autowired` on a single field, but it's about how this service might evolve in time and how we lay the foundation for the service.